### PR TITLE
[Doc] Add documentation for How to use prefixed service in custom rule

### PR DESCRIPTION
**Pull request by samsonasik**
ref https://github.com/rectorphp/rector/issues/6414#issuecomment-841138704

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: docs/how_to_use_prefixed_service_in_custom_rule.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+// rector.php
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    // ...
+
+	$services = $containerConfigurator->services();
+	$services->load('Symplify\\PackageBuilder\\', __DIR__ . '/vendor/symplify/package-builder/src');
+
+    // ...
+};
```
Comment (TomasVotruba) [inline docs/how_to_use_prefixed_service_in_custom_rule.md:38]: Here config can be improved instead.
Even more generic and Symfony best practice is to register service manually.

That can be used in any PHP class case

Comment (samsonasik) [reply docs/how_to_use_prefixed_service_in_custom_rule.md:38]: implemented https://github.com/rectorphp/rector-src/pull/33/commits/b571141c815ee70ab939fb512e33533dd5f028fe

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Thank you 😊

**Comment by samsonasik** [comment]
@TomasVotruba this new doc file seems not added to https://github.com/rectorphp/rector/tree/main/docs

**Comment by samsonasik** [comment]
new file seems not added in GHA scoped https://github.com/rectorphp/rector-src/runs/2586174810?check_suite_focus=true

**Comment by TomasVotruba** [comment]
Ah, I see. The /docs directory should be moved to /target-repository so it's publishes too

**Comment by samsonasik** [comment]
GHA scoped checkout possibly overlapped with new file added

**Pull request merged**